docs/servo_power_test/src/config.ts

```diff
@@ -4,17 +4,41 @@ import {ServoController} from './servo_controller';
 import {TestRunner} from './test_runner';
 import {Ui} from './ui';
 
+export class IterationData {
+  public powerDataList: Array<PowerData>;
+  public annotationList: AnnotationDataList;
```

Comment:
Can you make those two properties private? doing a similar thing (implementing getter / setter) as extractData does will make the encapsulation better ;)